### PR TITLE
Made skip nav

### LINKED ISSUE #3912: Skip Navigation -- quick tab/enter option
**What problem are you trying to solve with Submitty**
keyboard/screenreader users often want to quickly skip the sidebar (esp when it contains > 10 menu items) and jump quickly into the interesting items within the content portion of the page.

**Describe the way you'd like to solve this problem**
Let's make the first item on the side bar be 'skip navigation'.
But let's have it be hidden/not visible, EXCEPT when it has focus. 
When it has focus, it should be a visible option, probably overlaid on top of the side bar, towards the top.  

When you first load the page it's not visible (nothing on the page has focus yet).
When you hit tab the first time, the skip navigation now has focus, so it's visible.  
If you hit tab a second time, focus moves to the first thing on the side bar, skip navigation disappears again.
Or instead if you have focus on skip navigation and press enter, focus moves to the first content link within the page.

### PR DESCRIPTION
**Pull request by mktung**
Closes #3912

Made a skip to main content button that becomes visible to non-screen-reader users on focus.

### CODE REVIEW

File: site/app/templates/GlobalHeader.twig [hunk 1]
```diff
@@ -77,6 +77,7 @@
             </div>
             {% endif %}
             <div class="row" id="push">
+                <a id="skip-nav" href="#nav-positioner" tabindex="1">Skip to main content</a>
```
Comment (pbailie) [inline site/app/templates/GlobalHeader.twig]: Use of `tabindex` greater than zero is nearly always advised against in web accessibility standards.  q.v. https://webaim.org/techniques/keyboard/tabindex

Please remove the `tabindex` property and ensure tab order by logical placement of the link in HTML.

**Comment by bmcutler** [comment]
Yay!  I like this.  But there's a hiccup on the discussion forum page.  I think it's because that page (currently, as of PR #3859) starts with focus on the breadcrumb instead of requiring a tab to get there.

So please, @mktung  @xprilion coordinate who will get skip nav working on the forum page. 

Thanks

File: site/app/templates/GlobalHeader.twig (hunk 1 above)
Comment (mktung) [reply site/app/templates/GlobalHeader.twig]: I've made the changes

**Comment by mktung** [comment]
@xprilion I removed a line in the forum js that autofocused the page to the breadcrumb on load. If this was crucial, please let me know.

**Pull request merged**